cmd/start.go

```diff
@@ -63,17 +61,17 @@ func Start(repoName string, tp node.Type) *cobra.Command {
 				opts = append(opts, node.WithRemoteClient(protocol, ip))
 			}
 
-			headHash := cmd.Flag(headHash).Value.String()
-			if headHash != "" {
-				opts = append(opts, node.WithHead(tmbytes.HexBytes(headHash)))
+			genesisHash := cmd.Flag(genesis).Value.String()
+			if genesisHash != "" {
+				opts = append(opts, node.WithGenesis(genesisHash))
 			}
 
 			return start(cmd, tp, repo, opts...)
 		},
 	}
 
 	cmd.Flags().String(cfgAddress, "", "Indicates node to connect to the given remote core node")
-	cmd.Flags().String(headHash, "", "known header hash with which to initialize the node")
+	cmd.Flags().String(genesis, "", "Hex encoded block hash. Starting point for header synchronization.")
```

Comment:
this should also be clearer: Is it the genesis or some starting point?

Reply:
Let's think of how to name it. It can be genesis and also be just a random height to start syncing from. So technically, it would be even possible for FullNode without a core to sync the chain from height 2 or 2000. This is not a FullNode mb already but almost, because of the border between Full/Partial Storage Node.

Reply:
`Head` is also a bad name.

Reply:
Suggestions:
- trusted header hash
- checkpoint hash

Reply:
trusted header hash